dig/graph.go

```diff
@@ -128,11 +129,20 @@ func (g *graph) RegisterAll(cs ...interface{}) error {
 	return nil
 }
 
-// Reset the graph by removing all the registered nodes
```

Comment:
These need to be removed, because they appear in there different places. And as abg pointed out, these don't make it into GoDoc